agent/stats/utils_windows.go

```diff
@@ -42,3 +41,18 @@ func dockerStatsToContainerStats(dockerStats *types.StatsJSON) (*ContainerStats,
 		networkStats:      networkStats,
 	}, nil
 }
+
+// Safe return of storage stats based on Docker Stats (Windows)
+func getStorageStats(dockerStats *types.StatsJSON) (uint64, uint64) {
+	if dockerStats == nil {
+		return uint64(0), uint64(0)
+	}
+	if dockerStats.StorageStats.ReadSizeBytes == 0 && dockerStats.StorageStats.WriteSizeBytes == 0 {
```

Comment:
we should also check 
`if dockerStats.StorageStats == nil`

Reply:
see the above links to moby's implementation of the windows daemon.  It looks like they initialize all the structs or return nil for dockerStats.  So it's impossible to have a dockerstats without a StorageStats.